app/src/org/commcare/navdrawer/BaseDrawerActivity.kt

```diff
@@ -140,4 +140,8 @@ abstract class BaseDrawerActivity<T> : CommCareActivity<T>() {
     protected fun closeDrawer() {
         drawerController?.closeDrawer()
     }
+
+    fun openDrawer() {
+        drawerController?.openDrawer()
```

Comment:
can we soft assert using Logger.exception that drawerController is non null here. Just so that if we start calling this before `drawerController` is initialized, we should get to know about it from logs.